src/convert/convertContext.ts

```diff
@@ -18,17 +18,18 @@ import { WriteInfo, WriterFormat } from './types';
 abstract class ConvertTransactionFinalizer<T> {
   protected abstract transactionState: T;
 
-  public setState(props: (state: T) => void): void {
-    props(this.transactionState);
-  }
-
   public get state(): T {
     return this.transactionState;
   }
 
+  public setState(props: (state: T) => void): void {
+    props(this.transactionState);
+  }
+
   public abstract finalize(defaultDirectory?: string): Promise<WriterFormat[]>;
 }
 
+// must remain an export because vscode reaches in and imports it
```

Comment:
fixed that with [here](https://github.com/forcedotcom/salesforcedx-vscode/pull/4491). Feel free to not export.